pom.xml

```
@@ -1202,7 +1202,7 @@
                   <version>${maven.version}</version>
                 </requireMavenVersion>
                 <requireJavaVersion>
-                  <version>${java.version}</version>
+                  <version>[1.8, 1.9)</version>
```

```suggestion
                  <version>[${java.version}]</version>
```
as per the definition `[1.0] -> x == 1.0`
and also more likely to raise an issue/force a version reconsideration if `java.version` gets updated in the future

---

tried `[${java.version}]`, it does not work.
`[${java.version}]` will enforce the version to be exact 1.8, not even "1.8.0_sometag"  and our github IC failed previously under `[${java.version}]`

---

:(